src/css/styles/_top.scss

```diff
@@ -41,11 +46,34 @@
   padding: 1rem;
 }
 
+.l-center-vertical {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
```

Comment:
Should we have separate file for layouts. We have one for helpers.

- [ ] separate file for layouts